sdk/storage/azure-storage-blob-changefeed/azure/storage/blob/changefeed/_change_feed_client.py

```diff
@@ -25,10 +25,12 @@ class ChangeFeedClient(object):  # pylint: disable=too-many-public-methods
     :param credential:
```

Comment:
We should be updating the type hint for these as well. Since its a param, its listed in the function porotype below.

This applies to most of the files in this review (sorry!). Anywhere we document this, we should also update the type hint to match what you have in `base_client`. I guess that means PyLint disable and all sadly.

Reply:
Good point! Will get on that now 😄